### PR TITLE
gh-309 added the ui

### PR DESCRIPTION
**Pull request by DmitryBespalov**
- Snackbar ui
- Adjusting position according to the tab view visibility

Screen Recording 2020-06-24 at 17 19 35 mov

### CODE REVIEW

File: Multisig/UI/Assets/AssetsView.swift [hunk 1]
```diff
@@ -14,6 +14,9 @@ struct AssetsView: View {
 
     @State var selection: Int? = 0
 
+    @ObservedObject
+    var viewState: ViewState = App.shared.viewState
```
Comment (sche) [inline Multisig/UI/Assets/AssetsView.swift:18]: nice idea

File: Multisig/UI/UI Library/Overlays/SnackbarView.swift [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+            .offset(y: self.isPresented ?
+                -self.bottomSpacing : SnackbarViewMetrics.offscreenOffset)
+        }
+        .opacity(isPresented ? 1 : 0)
+        .animation(.spring())
+    }
+
+}
+
+struct SnackbarView_Previews: PreviewProvider {
```
Comment (sche) [inline Multisig/UI/UI Library/Overlays/SnackbarView.swift:50]: So did you manage to fix preview? Mine is still not working.

File: Multisig/UI/UI Library/Tab Views/Bottom Tab View/BottomTabView.swift [hunk 3]
```diff
@@ -44,14 +44,18 @@ struct BottomTabView<SelectionValue: Hashable>: View {
...
     }
 
-    let bigScreenHeight: CGFloat = 812 // 5.8 inch phones and higher
-    let bigScreenBarHeight: CGFloat = 84
-    let smallerScreenBarHeight: CGFloat = 64
+}
 
-    var barHeight: CGFloat {
-        UIScreen.main.bounds.height >= bigScreenHeight ? bigScreenBarHeight : smallerScreenBarHeight
+// extracted from the generic struct because Swift does not support
```
Comment (sche) [inline Multisig/UI/UI Library/Tab Views/Bottom Tab View/BottomTabView.swift:49]: smth new

**Review by sche — APPROVED** [review]

File: Multisig/UI/UI Library/Overlays/SnackbarView.swift (hunk 2 above)
Comment (DmitryBespalov) [reply Multisig/UI/UI Library/Overlays/SnackbarView.swift:50]: ah, no, it doesn't work

File: Multisig/UI/UI Library/Tab Views/Bottom Tab View/BottomTabView.swift (hunk 3 above)
Comment (DmitryBespalov) [reply Multisig/UI/UI Library/Tab Views/Bottom Tab View/BottomTabView.swift:49]: 🤷

**Review by MouazAlzahabi — APPROVED** [review]

**Pull request merged**